lib/middleware/metrics.rb

```diff
@@ -28,6 +28,18 @@ def is_private_ip?(env)
       !!(ip && ip.to_s =~ PRIVATE_IP)
     end
 
+    def is_trusted_ip?(env)
+      if
+        GlobalSetting.prometheus_trusted_ip_whitelist_regex.length == 0
+        nil
```

Comment:
I would return false here not nil, also the newline here next to the if needs to move up